### ISSUE TITLE
Ranged Collector doesn't collect items even when they would fit in its inventory

### ISSUE THREAD

**Issue by homemmakako** [Ellpeck/ActuallyAdditions#470]
R̶a̶n̶g̶e̶d̶ ̶C̶o̶l̶l̶e̶c̶t̶o̶r̶ ̶w̶i̶l̶l̶ ̶s̶t̶o̶p̶ ̶c̶o̶l̶l̶e̶c̶t̶i̶n̶g̶ ̶i̶t̶e̶m̶s̶ ̶a̶s̶ ̶s̶o̶o̶n̶ ̶a̶s̶ ̶a̶l̶l̶ ̶i̶t̶s̶ ̶s̶l̶o̶t̶s̶ ̶w̶e̶r̶e̶ ̶u̶s̶e̶d̶,̶ ̶e̶v̶e̶n̶ ̶i̶f̶ ̶n̶o̶t̶ ̶e̶n̶t̶i̶r̶e̶l̶y̶.̶ ̶F̶o̶r̶ ̶i̶n̶s̶t̶a̶n̶c̶e̶,̶ ̶i̶f̶ ̶y̶o̶u̶ ̶p̶u̶t̶ ̶o̶n̶e̶ ̶s̶i̶n̶g̶l̶e̶ ̶p̶i̶e̶c̶e̶ ̶o̶f̶ ̶c̶o̶b̶b̶l̶e̶s̶t̶o̶n̶e̶ ̶i̶n̶ ̶e̶v̶e̶r̶y̶ ̶s̶l̶o̶t̶ ̶i̶t̶ ̶w̶o̶n̶'̶t̶ ̶p̶i̶c̶k̶ ̶u̶p̶ ̶c̶o̶b̶b̶l̶e̶ ̶n̶e̶a̶b̶y̶,̶ ̶e̶v̶e̶n̶ ̶t̶h̶o̶u̶g̶h̶ ̶o̶n̶e̶ ̶s̶l̶o̶t̶ ̶s̶h̶o̶u̶l̶d̶ ̶b̶e̶ ̶a̶b̶l̶e̶ ̶t̶o̶ ̶h̶o̶l̶d̶ ̶u̶p̶ ̶t̶o̶ ̶6̶4̶ ̶i̶t̶e̶m̶s̶.̶

**Versions:**
MC: 1.10.2
Forge: 12.18.3.2185
AA: 1.10.2-r88

**Comment by Ellpeck** [comment]
I presume this only happens when you have, say, 59 cobblestone already in the collector and you are trying to pick up a single stack of, say, 20?
Because it isn't supposed to be able to split the item entity's item stack like that.

**Comment by homemmakako** [comment]
Was testing the same bug on 1.11, and it seems the same bug occurs. However I realized it is not how I described previously. Actually what happens is the collector won't pickup the items if they are stacked in groups that would not fit in any of the collector's slots. As an example, suppose you had every slot in the Ranged Collector with 32 cobble. If you throw 32 cobble or less nearby the  it **WILL** pick them up, but if you throw more than 32 items at once (so they are stacked) it won't pick them up, since every slot can only have 64 items at once and adding the 32 in the slot with any number greater than 32 would go beyond a full 64 items stack.
